<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.949, y1:0.96, x2:0, y2:0, stop:0.193182 rgba(104, 41, 41, 209), stop:0.931818 rgba(104, 0, 0, 209));
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="bar">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Bar Graph</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="scatter">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Scatter Graph</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pie">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Pie Graph</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="line">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Line Graph</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="prim_bt">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>MST Using Prims</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="kruskal_bt">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>MST Using Kruskal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bfs_dfs_bt">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>BFS DFS</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
